sandbox/apparmor/apparmor.go

```diff
@@ -51,6 +53,30 @@ const (
 	Full
 )
 
+func setupConfCacheDirs(newrootdir string) {
+	ConfDir = filepath.Join(newrootdir, "/etc/apparmor.d")
+	CacheDir = filepath.Join(newrootdir, "/var/cache/apparmor")
+
+	SystemCacheDir = filepath.Join(ConfDir, "cache")
+	exists, isDir, _ := osutil.DirExists(CacheDir)
```

Comment:
this should actually check the SystemCacheDir  (see original code)

Reply:
fixed with a force push